require('./app/main').catch(err => console.log(err));
